src/test/java/ioananadroflorea/tests/Tema5/HerokuAppTests.java

```diff
@@ -0,0 +1,65 @@
+package ioananadroflorea.tests.Tema5;
+
+import ioanaandroflorea.driver.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.util.List;
+
+public class HerokuAppTests {
+    public static void main(String[] args) {
+        goToBasicWebPageAndCheckElements();
+        elementAttributesExample();
+        locatorsFindByPage();
+        tableTestPage();
+    }
+
+    public static void goToBasicWebPageAndCheckElements() {
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        driver.get("https://testpages.herokuapp.com/styled/index.html");
+        WebElement firstLink = driver.findElement(By.className("basicpagetest"));
```

Comment:
I think that you should use id instead of className here. This is the exception ->
Exception in thread "main" org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"css selector","selector":".basicpagetest"}